### DISCUSSION TITLE
parameter "-n" - be sure plot done one by one ?

### DISCUSSION THREAD

**Discussion by casa10** [madMAx43v3r/chia-plotter#328] [General]
if you put paramater -n = 5, do the plot will be done one by one, that is to say when one is finished AND completely transfered on HDD, the second plot will start ? 
Thank you

**Comment by SebMoore** [comment]
That is almost correct: it will do them one by one, but it will do the transfers in the background so the plotter is not held up waiting for the transfer to complete.

**Reply by casa10** [reply]
So when one is finished to be created, the next begins and transfer of the plots finished is done in background ?

**Reply by SebMoore** [reply]
Yup!

**Reply by casa10** [reply]
thank you :)
